src/routes/(admin)/dashboard/+page.svelte

```
@@ -82,21 +84,17 @@
 		{#each stats_box as stat}
-			<div class="m-4 w-1/3 rounded-xl bg-white p-4">
-				<p class="font-semibold">{stat.title}</p>
+			<div
+				class="animate-fadeIn m-4 w-1/3 rounded-xl bg-white p-4 shadow-lg hover:scale-105 hover:shadow-2xl hover:shadow-2xl"
+			>
+				<p class="mb-2 border-b pb-2 font-semibold">{stat.title}</p>
 				<p class="text-sm">{stat.description}</p>
-				<div class="p-12 text-5xl">{stat.count}</div>
+				<div class="p-12 text-5xl font-semibold">{stat.count}</div>
 			</div>
 		{/each}
```

ESLint

[error] Each block should have a key

(svelte/require-each-key)